alma/alma-installments-prestashop
# Feature/mpp 528/make compilation module prestashop from docker to fix the version php

### Reason for change

Linked to https://linear.app/almapay/issue/DEX-411/%F0%9F%8F%83-i-need-make-a-docker-to-build-my-zip-module-in-specific-version-of

### Code changes

We want to build using docker, so that the build process is repeatable, on every machine.

### How to test

_As a reviewer, you are encouraged to test the PR locally._

You can run `make dist` locally, and check the content of the zip file generated;

### Checklist for authors and reviewers




### Non applicable
